skaffold.kind.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: skaffold/v4beta6
+kind: Config
+manifests:
+  kustomize:
+    paths:
+      - ./config/install-kind
+build:
+  artifacts:
+    - image: docker.io/substratusai/controller-manager
+      docker:
+        dockerfile: Dockerfile
+    - image: docker.io/substratusai/sci-kind
+      docker:
+        dockerfile: Dockerfile.sci-kind
+  local:
+    push: false
```

Comment:
how does this work, does kind automatically use the images built on the host? This shouldn't be working unless you do this: https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster

Reply:
found out why it works: https://skaffold.dev/docs/environment/local-cluster/